docs/tutorial/progress-bar.md

```diff
@@ -1,35 +1,56 @@
 # Progress Bar in Taskbar (Windows, macOS, Unity)
 
-On Windows a taskbar button can be used to display a progress bar. This enables
-a window to provide progress information to the user without the user having to
-switch to the window itself.
+## Overview
 
-On macOS the progress bar will be displayed as a part of the dock icon.
+On Windows, a taskbar button can be used to display a progress bar. This enables
+a window to provide progress information to the user without the need of switching to
+the window itself.
+
+![Taskbar Progress Bar][taskbar-progress-image]
+
+On macOS, the progress bar will be displayed as a part of the dock icon.
 
 The Unity DE also has a similar feature that allows you to specify the progress
 bar in the launcher.
 
-__Progress bar in taskbar button:__
+----
 
-![Taskbar Progress Bar][taskbar-progress-image]
-
-All three cases are covered by the same API - the `setProgressBar()` method
-available on instances of `BrowserWindows`. Call it with a number between `0`
-and `1` to indicate your progress. If you have a long-running task that's
-currently at 63% towards completion, you'd call it with `setProgressBar(0.63)`.
+All three cases are covered by the same API - the
+[`setProgressBar()`][setprogressbar] method available on an instance of
+`BrowserWindow`. To indicate your progress, call this method with a number
+between `0` and `1`. For example, if you have a long-running task that is
+currently at 63% towards completion, you would call it as
+`setProgressBar(0.63)`.
 
-Generally speaking, setting the parameter to a value below zero (like `-1`)
-will remove the progress bar while setting it to a value higher than one
-(like `2`) will switch the progress bar to intermediate mode.
+Setting the parameter to negative values (e.g. `-1`) will remove the progress
+bar, whereas setting it to values greater than `1` (e.g. `2`) will switch the
+progress bar to intermediate mode (100% complete).
```

Comment:
I think this was supposed to say "indeterminate" mode, but the typo persisted for a long time without anyone noticing.

See API doc: https://www.electronjs.org/docs/api/browser-window#:~:text=Change%20to%20indeterminate%20mode%20when%20progress%20%3E%201

Reply:
Exactly. API is clear on that